adhocracy4/polls/static/polls/PollManagement.jsx

```diff
@@ -43,6 +43,7 @@ class PollManagement extends React.Component {
   */
 
   getNewQuestion (label = '') {
+    console.log('I am so here')
```

Comment:
This should probably also go before merge?

Reply:
haha, ups :see_no_evil: